src/main/services/typeOfBusinessService.ts

```diff
@@ -30,4 +33,48 @@ export class TypeOfBusinessService extends GenericService {
             return Promise.reject(errorData);
         }
     }
+
+    async saveAcspData (session: Session, selectedOption: string): Promise<void> {
```

Comment:
Now that we have this method in service class.. All the contollers should call this method instead of posting or putting on their own. This will avoid changing in so many places again if tomorrow we decide to make another change in post/put methods.

Reply:
This is specific to the type of business screens, it can't be reused in the other screens as were not passing that data into the method

Reply:
Resolved this